.github/workflows/nightly.yml

```diff
@@ -0,0 +1,53 @@
+name: Nightly release
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - workflow
+
+jobs:
+  nightly:
+
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3.0.2
+      with:
+        fetch-depth: 1
+    
+    - name: Build with npm
+      uses: actions/setup-node@v3
+      with:
+        node-version: 10.15
```

Comment:
Why this old node version? Please add a comment if this is really required. Otherwise, I would propose to use v20 (LTS)

Reply:
Pretty sure h5ai as a whole got an issue building with newer versions of node, did you try to build the program yourself?

Reply:
Then a code comment could be added to make it clear. It might happen that someone activates dependabot...

Reply:
anyone who tried to build the app would know this 💀